packages/build-info/src/runtime/bun.test.ts

```diff
@@ -0,0 +1,27 @@
+import { beforeEach, expect, test } from 'vitest'
+
+import { mockFileSystem } from '../../tests/mock-file-system.js'
+import { NodeFS } from '../node/file-system.js'
+import { Project } from '../project.js'
+
+beforeEach((ctx) => {
+  ctx.fs = new NodeFS()
+})
+
+test('detects node when .node-version is present', async ({ fs }) => {
```

Comment:
```suggestion
test('detects node when bunfig.toml is present', async ({ fs }) => {
```
I think this test case description suffers from copy-pasta syndrome